mp/src/game/server/client.cpp

```diff
@@ -855,7 +855,8 @@ CON_COMMAND( give, "Give item to player.\n\tArguments: <item_name>" )
 	CBasePlayer *pPlayer = ToBasePlayer( UTIL_GetCommandClient() ); 
 	if ( pPlayer 
 		&& (gpGlobals->maxClients == 1 || sv_cheats->GetBool()) 
-		&& args.ArgC() >= 2 )
+		&& args.ArgC() >= 2
+		&& pPlayer->GetObserverMode() == 0 )
```

Comment:
I'd say use SPEC_NONE here, 0 is not very clear.

Reply:
There also may be a "IsObserver()" bool you could use instead that could be the most clear.